main.js

```diff
@@ -0,0 +1,74 @@
+// Tracks Board State //
+let gameBoardObj = {
+  1: [5, 4, 3, 2, 1],
+  2: [],
+  3: []
+}
+
+
+
+
+// Handles Opening Display of Game Board //
+let renderGame = function() {Object.values(gameBoardObj).map((game) => {
```

Comment:
you are using es6 let declaration with a self invoking function, decide if you want es6 and then go all the way with arrow functions as well.